gradle/shipkit.gradle

```diff
@@ -9,4 +9,6 @@ shipkit {
 afterEvaluate {
     //TODO let's create a property for this setting in 'shipkit' extension
     tasks.updateReleaseNotes.publicationRepository = "https://plugins.gradle.org/plugin/org.shipkit.java"
+
+    tasks.findByName(JavaBasePlugin.CHECK_TASK_NAME).dependsOn(validatePlugins)
```

Comment:
Typically in Gradle build script file you would see:

```
check.dependsOn validatePlugins
//or:
tasks.check.dependsOn validatePlugins
```

This way it is shorter and easier to read. In the plugin code, in java, we would be more explicit and use 'findByName'.

Build script files and plugin code are both optimized for maintainability. In build.gradle file readability gives most maintainability whereas in plugin code, the best is explicit calls with compile-time safety.